jaeger-crossdock/src/test/java/com/uber/jaeger/crossdock/resources/behavior/http/TraceBehaviorResourceTest.java

```diff
@@ -82,12 +82,10 @@ public static void setUpLogger() {
 
   @Before
   public void setUp() throws Exception {
-    port = System.getenv("TRACE_TEST_BEHAVIOR_PORT");
```

Comment:
looks like this variable is only used for testing but even then it's never set anywhere. Looks ok for removal. The crossdock tests are still passing.